src/lapack.jl

```diff
@@ -138,4 +139,112 @@ for (laqps, elty, relty) in ((:slaqps_, :Float32,    :Float32),
   end
 end
 
+for (geqp3rk, elty, relty) in ((:sgeqp3rk_,:Float32,:Float32),
+                               (:dgeqp3rk_,:Float64,:Float64),
+                               (:cgeqp3rk_,:ComplexF32,:Float32),
+                               (:zgeqp3rk_,:ComplexF32,:Float64))
```

Comment:
The second ComplexF32 should be ComplexF64

Reply:
Yes on my local I fixed that. Hang on let me push a few changes